articles/cosmos-db/create-mongodb-golang.md

```diff
@@ -0,0 +1,242 @@
+
+Azure Cosmos DB is Microsoft’s globally distributed multi-model database
+service. You can quickly create and query document, key/value, and graph
+databases, all of which benefit from the global distribution and horizontal
+scale capabilities at the core of Azure Cosmos DB.
+
+This quick-start demonstrates how to use an existing
+[MongoDB](https://docs.microsoft.com/en-us/azure/documentdb/documentdb-protocol-mongodb)
+app written in Golang and connect it to your Azure Cosmos DB database, which
+supports MongoDB client connections.
+
+In other words, your Golang application only knows that it’s connecting to a
+database using MongoDB APIs. It is transparent to the application that the data
+is stored in Azure Cosmos DB.
+
+We’ll cover:
+
+* Prerequisites for this tutorial
+* Creating and connecting to an Azure Cosmos DB account
+* Setting up your application
+* Connect to an Azure Cosmos DB account
+* CRUD Operations
+
+### Prerequisites for the Golang tutorial
+
+1.  Basic knowledge of [GO ](https://golang.org/)language
+1.  Azure subscription. (If you don’t have an Azure subscription, create a [free
+account](https://azure.microsoft.com/free/?WT.mc_id=A261C142F) before you
+begin.)
+1.  IDE — [Gogland](https://www.jetbrains.com/go/) by Jetbrains or [Visual Studio
+Code](https://code.visualstudio.com/) by Microsoft or [Atom](https://atom.io/)
+
+### Creating and connecting to an Azure Cosmos DB account
+
+1.  In a new window, sign in to the [Azure portal](https://portal.azure.com/).
+1.  In the left menu, click **New**, click **Databases**, and then click **Azure
+Cosmos DB**.
+
+![](https://cdn-images-1.medium.com/max/800/1*e-QMvThW-2QZ3hEH3VvUQQ.png)
+
+3. In the **New account** blade, specify the desired configuration for the Azure
+Cosmos DB account.
+
+With Azure Cosmos DB, you can choose one of four programming models: Gremlin
+(graph), MongoDB, SQL (DocumentDB), and Table (key-value).
+
+In this quick start we’ll be programming against the MongoDB API so you’ll
+choose **MongoDB** as you fill out the form. But if you have graph data for a
+social media mes New app, document data from a catalog app, or key/value (table)
+data, realize that Azure Cosmos DB can provide a highly available,
+globally-distributed database service platform for all your mission-critical
+applications.
+
+Fill out the New account blade using the information in the screenshot as a
+guide . You will choose unique values as you set up your account so your values
+will not match the screenshot exactly
+
+![](https://cdn-images-1.medium.com/max/800/1*XdFlpeNKLe7o3FOtCZ17AQ.png)
+<span class="figcaption_hack">Create Azure Cosmos DB Account</span>
+
+4. Click **Create** to create the account.
+
+5. On the toolbar, click **Notifications** to monitor the deployment process.
+
+![](https://cdn-images-1.medium.com/max/800/1*WTGVkPwkSTxjUzKm2u09Eg.png)
+<span class="figcaption_hack">Check Progress of Azure Cosmos DB Deployment</span>
```

Comment:
In preview mode, this text is not aligned correctly with the image @mimig1 can you help here